### ISSUE TITLE
Syntax error on coparison with a bind variable on the left

### ISSUE THREAD

**Issue by gdsotirov** [buntatsu/cdt-proc#9]
There is a syntax error on queries in which there is a comparison with a bind variable on the left (see screen shot below). There is no such error when the bind variable is on the right of the expression.

![ecdt_proc_bind_var_cmpr_err](https://cloud.githubusercontent.com/assets/955033/10487361/3fb9c5b4-729c-11e5-86e0-43381d92e31f.png)

The example code is:

```sql
EXEC SQL INCLUDE sqlca.h;

int main(int argc, char * argv[])
{
  char test[] = 'ABC';

  EXEC SQL SELECT *
             FROM dual
            WHERE :test <> 'TEST';

  EXEC SQL SELECT *
             FROM dual
             WHERE 'TEST' <> :test;
}
```

**Comment by gdsotirov** [comment]
Unfortunately, it doesn't work for me. I've upgraded Eclipse to 4.5.1 and CDT to 8.8, then installed [release 3.0.0](https://github.com/buntatsu/cdt-proc/releases/tag/3.0.0) following [the procedure](https://github.com/buntatsu/cdt-proc), but I still have the same error both in the example source and real sources. I've also rebuilt the C/C++ index manually, but there was no change.

![eclipse_cdt_proc-version](https://cloud.githubusercontent.com/assets/955033/10635621/b40daf78-7802-11e5-8d71-f8bda9151738.jpg)

What am I missing and what could be the problem?

**Comment by buntatsu** [comment]
I just commited the source. It is not yet released.
Please wait for version 3.1.0.

**Comment by buntatsu** [comment]
I released version [3.1.0](https://github.com/buntatsu/cdt-proc/releases/tag/3.1.0)
Please try it.

**Comment by gdsotirov** [comment]
Yep, I confirm it fixed in [release 3.1.0](https://github.com/buntatsu/cdt-proc/releases/tag/3.1.0). I must have overlooked the release notes for [release 3.0.0](https://github.com/buntatsu/cdt-proc/releases/tag/3.0.0), but they are clearly stating the purpose of the release and commit 964813f wasn't included.

**Comment by buntatsu** [comment]
You're welcome.
Thank you as always pointed out.
